automation/jleechanorg_pr_automation/openai_automation/test_codex_comprehensive.py

```
@@ -0,0 +1,351 @@
+    pytest automation/openai_automation/test_codex_comprehensive.py -v
+"""
+
+import asyncio
+import os
+import sys
+from unittest.mock import AsyncMock, Mock
+
+import aiohttp
+import pytest
+
+project_root = os.path.dirname(os.path.abspath(__file__))
```

Ruff (0.14.8)

`os.path.dirname()` should be replaced by `Path.parent`

(PTH120)